src/evergreen/base.py

```diff
@@ -58,29 +66,33 @@ def evg_date_attrib(attrib_name):
 class _BaseEvergreenObject(object):
     """Common evergreen object."""
 
-    def __init__(self, json, api):
-        """
-        Create an instance of an evergreen task.
-        """
+    def __init__(self, json: Dict, api: EvergreenApi) -> None:
```

Comment:
IMO we should do a find and replace: `json: Dict` --> `json: Dict[str, Any]`

Reply:
done.